code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -25,6 +25,7 @@
 	if(icon_state == "[initial(icon_state)]open")
 		opened = TRUE
 	update_icon()
+	RegisterSignal(src, COMSIG_PARENT_ATTACKBY, .proc/tear_manifest)

```

Comment:
Why are you registering a proc to the src object? There is nothing wrong with just calling tear_manifest from the proc.

Reply:
probably left over from when there was meant to be a new component for this